### PR TITLE
docs: add skip_null_values in configuration, operations and upgrade-g…

### PR DESCRIPTION
**Pull request by mdieudonne**
https://github.com/api-platform/docs/issues/1616

This documents the new default value for skip_null_values.

### CODE REVIEW

File: core/operations.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ will be automatically added.
 
 Note: The `#[GetCollection]` attribute is an alias for `#[Get(collection: true)]`
 
+Note: Default value of the `skip_null_values` property is `true`, which means that null values will be omitted when serializing.
```
Comment (alanpoulain) [inline core/operations.md:68]: I find it weird to have this note here.

File: core/upgrade-guide.md [hunk 2]
```diff
@@ -15,6 +15,14 @@ api_platform:
 This will be the default value in 3.0, in 2.7 it's left to `true` so that nothing breaks by updating.
 By doing so you won't get access to legacy services and this will probably break things on code using `api-platform/core:2.6`.
 
+In 3.0 the default value of the `skip_null_values` property will be `true`, which means that from now on null values will be omitted when serializing.
```
Comment (alanpoulain) [inline core/upgrade-guide.md:18]:
```suggestion
In 3.0 the default value of the `skip_null_values` property is `true`, which means that from now on null values are omitted when serializing.
```

Comment (dunglas) [reply core/upgrade-guide.md:18]: maybe can we add somewhere that it's to be in conformance with the JSON Merge Patch RFC and that changing this setting could make the API non-compliant?

**Review by dunglas — APPROVED** [review]

**Review by alanpoulain — APPROVED** [review]

**Comment by alanpoulain** [comment]
Thanks @mdieudonne.

**Pull request merged**